app/controllers/users_controller.rb

```diff
@@ -93,7 +93,11 @@ def update_language_settings
 
   def request_destroy
     set_tabs("account")
-    if @user.email?
+
+    if destroy_request_in_progress
```

Comment:
Maybe we want to make this a predicate method too, `destroy_request_in_progress?`?